### DISCUSSION TITLE
Configuration of DSL + 2 x 5G

### DISCUSSION THREAD

**Discussion by Olli2k** [Ysurac/openmptcprouter#3600] [Q&A] [answered]
Hey there, 
first of all I would like to thank you (0JS48973EM2319601 not only in words!) for this project! You saved me from a 200mbit Internet connection on the countryside.

Now about my topic. I have got the following setup:
Fritzbox 7690 - Telekom DSL DL: 200 / UL: 40, Upling 2,5G
Mikrotik Chateau 5G - Telekom 5G DL: 500 / UL: 110, Uplink 1G
ZTE MTU5002 5G - Telekom 5G DL: 500 / UL: 110, Uplink 1G

OMR is running on a Minisforum MS-01 as Proxmox container. Here I'm using the SFP+ 10G connections.

As VPS there is hetzner and Avoro. Currently I'm running openmptcprouter v0.61rc3-6.6 (r0+27346-c7ba5574f5 and for the VPS 0.1031-test 6.6.36-x64v3-xanmod1 for Avoro. For Hetzner VPS it is v4, what is wondering me, because I used same script URLs for installation/updating.

Ok, from a simple addition speed should be about DL: 500 + 500 + 200 = 1200 and UL: 40 + 110 + 110 = 220.
I'm aware, taht these values are not reachable in connection speed. But due to the fact, that I tried nearly every combination of Default proxy, SQM, SQM Rotate over the last year, I'm wondering if my speed results are maximum I can get out of my configuration. In General Download is about 650mbit and Upload is about 165mbit. The most confusing fact for me, is that when I switch off one of my 5G connection speed remains nearly the same. When I activate all 3 wans, speed is about: DSL 180, 5G 1: 240, 5G 2: 210. When I turn off one of the 5 G wan it is about DSL: 180 and 5G 450-500. To ensure it is not based on the provider I did simultaneous test with 2 phones (same provider), when running the speed tests on my OMR.

To exclude my home lan, as reason for the slodown of my 5 G connection, a small overview of my setup:
Main Switch: Mikrotik 24 SFP+ and not used QSFP. From there is an direct Uplink to local OMR install. 2 time 10G. Speed checked with iperf. Then I have a uplink of 2.5 G to my DSL connection. And another 10G Uplink to another Mikrotik Switch with 8 1G Ports. Two of them are connected to my 5G router. Every router has a dedicated vlan and all connection were checked with iperf. There are no speed issues.

So is there any recommendation of settings for my setup? Thx in advance.

**Comment by Ysurac** [comment, marked as answer]
The kernel v1 to v4 is based on the psABI, so the model of CPU.
Your 5G connections are using same ISP and same antenna, from your test it seems that it's limited by this. The used frequencies can be different from modem and phone.

For speed tests, you can use ```omr-test-speed``` (and ```omr-test-speed eth1``` to test speed via eth1) on the router via SSH.

**Comment by Olli2k** [comment]
Seems you are right. Did several tests on console at the same time for wan2 and wan3. Total speed was 55m for one connection and when the second test started speed was shared between the interfaces. So another provides is needed for the second 5G connection.. shit ;).
